docker/orb.yaml

```diff
@@ -44,8 +44,9 @@ commands:
       - run:
           name: docker build
           command: |
-            docker build \
+            DOCKER_BUILDKIT=1 docker build \
```

Comment:
You can also [use the `environment` field to set this](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-step). I genuinely have no idea what the difference would be, but my assumption would be if they're providing a specific method for this it's probably worth using?